m4/ltp-sendmmsg.m4

```diff
@@ -0,0 +1,8 @@
+dnl SPDX-License-Identifier: GPL-2.0-or-later
+dnl Copyright (c) 2019 Google, Inc.
+
+dnl LTP_CHECK_SENDMMSG
+dnl ----------------------------
+AC_DEFUN([LTP_CHECK_SENDMMSG],[
+AC_CHECK_FUNCS(sendmmsg,,)
+])
```

Comment:
We decided to add these checks to the AC_CHECK_FUNCS() in the configure.ac instead. Can you please rebase the patchset on the top of the http://patchwork.ozlabs.org/patch/1078150/ once it gets merged?